resources/db/opiskeluoikeudet/select_count_by_koulutustoimija.sql

```diff
@@ -0,0 +1 @@
+SELECT COUNT(DISTINCT oid) FROM opiskeluoikeudet WHERE koulutustoimija_oid = ?
```

Comment:
Tämä taitaisi mennä myös ilman distinctiä kun on tuo oid on primary key ja siitä tulee unique constraint oid-kentälle, mutta eipä tuosta haittaakaan ole. Taisi olla että vasta ihan äskettäin lisättiin tuo primary key ja aiemmin oli mahdollisia duplikaatit.